_config/features_comparison.yml

```diff
@@ -0,0 +1,241 @@
+features:
+  - name: "ACID"
+    jta: true
+    jts: true
+    xts: true
+    comp: false
+    rts: true
+    stm: true
+    blacktie: true
+    arjuna: true
+    txoj: true
```

Comment:
TXOJ is much more than just a concurrency framework: it provides the isolation aspects that are missing from ArjunaCore and the resultant is an ACID compliant framework. If we want to suggest that TXOJ only provides the I in ACID then we should make sure people understand that ArjunaCore only provides the A and D.